oomox_gui/theme_model.py

```diff
@@ -324,7 +324,7 @@ def merge_theme_model_with_base(whole_theme_model, base_theme_model, plugin_mode
         'key': 'TERMINAL_THEME_EXTEND_PALETTE',
         'type': 'bool',
         'fallback_value': False,
-        'display_name': _('Extend palette with more lighter/darker colors'),
+        'display_name': _('Extend Palette with More Lighter/Darker Colors'),
```

Comment:
and this one

It Just Looks Too Unusual When I See Quite a Long Sentence Written All Capitalized

Reply:
mb just the sentence itself should be rewritten to be shorter